### ISSUE TITLE
add `page.driver.pause` as  additional element  for DebuggerMethods of `Lint/Debugger` rubocop 

### ISSUE THREAD

**Issue by Ivanov-Anton** [rubocop/rubocop#10859]
## Is your feature request related to a problem? Please describe.

The problem is when I perform debugging with `page.driver.pause` of Cuprite this line should be detected by robocop.

## Describe the solution you'd like

I suggest adding this `page.driver.pause` debugger to DebuggerMethods Configurable attributes of Lint/Debugger cop

**Comment by dvandersluis** [comment]
You should be able to handle this in your `.rubocop.yml` file. Try adding the following:

```yaml
Lint/Debugger:
  DebuggerMethods:
    Cuprite:
      - page.driver.pause
```

**Comment by dvandersluis** [comment]
I have opened #10860 to make it more clear how you can add your own debugger methods for the cop, but there are no code changes needed here.

**Comment by Ivanov-Anton** [comment]
@dvandersluis Thank you for helping I tried it previously but It is not working.

## Step to reproduce:
1. create file `test_spec.rb`
```
# frozen_string_literal: true

RSpec.describe '' do
  it 'should' do
    page.driver.pause
  end
end
```
2. add the following code in the `test_spec.rb` file
3. add the following code in the `.rubocop` config file
```
Lint/Debugger:
  DebuggerMethods:
    Cuprite:
      - page.driver.pause
```
4. rub `bundle exec rubocop test_spec.rb`


## Expected behaviour:
```
test_spec.rb:5:5: W: Lint/Debugger: Remove debugger entry point page.driver.pause.
    page.driver.pause
```

## Actual behaviour:
```
Inspecting 1 file
.

1 file inspected, no offenses detected
```

## Additional info:
robocop (1.32.0)
ruby-3.1.2

**Comment by dvandersluis** [comment]
@Ivanov-Anton thank you, I can replicate now. The cop is not handling method chains properly. I'll have a PR shortly.
